index.js

```diff
@@ -1074,11 +1202,16 @@ const createFocusTrap = function (elements, userOptions) {
         }
 
         state.paused = paused;
+
         if (paused) {
           const onPause = getOption(options, 'onPause');
           const onPostPause = getOption(options, 'onPostPause');
           onPause?.();
 
+          if (config.isolateSubtrees) {
+            setSubtreeIsolation(false);
+          }
```

Comment:
Might be necessary to not have this tied so closely to pausing/unpausing per my previous comment, and so we might want to remove this and replace it with more _precise_ subtree isolation calls...